recipes/simdjson/0.4.7/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.9)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

Cmake 3.1 is enough here.